frontend/pages/SoftwarePage/components/forms/AdvancedOptionsFields/_styles.scss

```
@@ -1,5 +1,4 @@
 .advanced-options-fields {
-  display: flex;
-  flex-direction: column;
-  gap: $pad-medium;
+  @include flex-column-16px-gap;
+  width: 100%;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)